app/services/gtfs_graph.rb

```diff
@@ -193,7 +202,7 @@ def load_tl_routes
 
   def load_tl_operators
     # Operators
-    log "  operators"
+    log '  operators'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.